backfill.py

```diff
@@ -23,24 +23,40 @@
 
 KEY_NAME = "name"
 KEY_DATETIME = "date"
+KEY_COMMIT = "commit"
+KEY_ARCHITECTURE = "architecture"
 
 DATETIME_FORMAT = "%Y.%m.%d"
 
+FENIX_CHANNEL_TO_PKG = {
+    'nightly': 'org.mozilla.fenix',
+    'beta': 'org.mozilla.firefox.beta',
+    'release': 'org.mozilla.firefox',
+    'debug': 'org.mozilla.fenix.debug'
+}
+
 
 def parse_args():
     parser = argparse.ArgumentParser(description=DESCRIPTION)
 
     parser.add_argument("path_to_startup_script",
                         help="Path to the measure_start_up.py script needed to gather startupperformance metrics")
-    parser.add_argument("package_id", help="Package id associated with the apk to install / uninstall")
-    parser.add_argument("start", type=lambda date: datetime.strptime(date, DATETIME_FORMAT),
+    parser.add_argument("build_type", choices=["nightly", "beta", "release", "debug"],
```

Comment:
issue (blocking): this script seems unintuitive to use. For example,
```
$ python3 backfill.py --help
usage: backfill.py [-h] -t {nightly,commitsDate,commitsRange} [--startdate STARTDATE] [--enddate ENDDATE]
                   [--startcommit STARTCOMMIT] [--endcommit ENDCOMMIT] [--repo_branch REPO_BRANCH]
                   [--git_remote_name GIT_REMOTE_NAME] [-r REPOSITORY_PATH] [-c]
                   path_to_startup_script {nightly,beta,release,debug} {armeabi-v7a,arm64-v8a-}
```

Some thoughts that jump out at me:
- Do we need both commitsDate and commitsRange? What use case do we have for commitsDate that wouldn't be covered by commitsRange?
- Why do we need `--git_remote_name`? Couldn't we just check the branch out locally and make the script simpler? Is this for commitsDate?
- Why do we need `--repo_branch`? Wouldn't passing a specific commit range take care of this? Is this for commitsDate?
- It seems like `-t` should be a positional argument given that it's required. I think this would reduce some cognitive overhead.

If I understand this correctly, it It seems like we may be introducing a lot of complexity to support commitsDate and I'm not convinced that we need it.